core/types/src/attributes.rs

```diff
@@ -145,7 +145,7 @@ bitflags! {
         /// PKRU(Protection Keys Register Userspace) state
         const PKRU = SGX_XFRM_PKRU as u64;
         /// AMX(Advanced Matrix Extensions), including XTILEDATA(0x40000) and XTILECFG(0x20000)
-        const AMX = SGX_XFRM_LEGACY as u64;
+        const AMX = SGX_XFRM_AMX as u64;
```

Comment:
This was noticed tracking down the build failure in https://github.com/mobilecoinfoundation/attestation/actions/runs/5391200227/jobs/9787680237